Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -275,6 +276,75 @@ extension VisualInstruction.Component: Codable {
         case lane
     }
     
+    /**
+     :nodoc:
+     Subtype that provides more context about the component guidance view that may help in visual
+     markup and display choices.
+     */
+    public enum SubType: String, Codable, CaseIterable {
+        
+        /**
+         Junction view. Bird’s-eye artist’s rendition view of the overhead signage and
+         preferred road lane arrow on motorways where the road bifurcates into 2 or
+         more motorway trunk roads.
+         */
+        case jct = "jct"
+        
+        /**
+         Advanced 2D signboard (vendor enhanced detailed signboard).
+         */
+        case signboard = "signboard"
+        
+        /**
+         Service area/parking area guide map. Vertical artist’s rendition guide map of an SAPA rest area
+         showing various facilities icons such as restaurants, restrooms and parking areas.
+         The scale is approximately 1 to 5K.
+         */
+        case sapaGuideMap = "sapaguidemap"
+        
+        /**
+         Service area/parking area. Bird’s-eye artist’s rendition view of the overhead
+         signage and preferred road lane arrow at a rest area ramp where the route
+         leaves the main road.
+         */
+        case sapa = "sapa"
```

Comment:
The “SAPA” abbreviation is specific to Japan. Consider renaming to `serviceArea` for discoverability and consistency with `RestStop.StopType.serviceArea`.